static/app/views/dashboardsV2/widget/metricWidget/index.tsx

```diff
@@ -83,11 +83,16 @@ class MetricWidget extends AsyncView<Props, State> {
     }
 
     const orgSlug = organization.slug;
-    const projectSlug = this.project.slug;
 
     return [
-      ['metricMetas', `/projects/${orgSlug}/${projectSlug}/metrics/meta/`],
-      ['metricTags', `/projects/${orgSlug}/${projectSlug}/metrics/tags/`],
+      [
+        'metricMetas',
+        `/organizations/${orgSlug}/metrics/meta/?project=${this.project.id}`,
```

Comment:
@priscilawebdev not sure if this is the correct way to add a query param.

Reply:
this works, but if you prefer to use queries, you can write it like this:

```
const projectId = this.project.id;

    return [
      [
        'metricMetas',
        `/organizations/${orgSlug}/metrics/meta/`,
        {query: {project: projectId}},
      ],
      [
        'metricTags',
        `/organizations/${orgSlug}/metrics/tags/`,
        {query: {project: projectId}},
      ],
    ];
```